time stamp
string frame_id
time start_stamp
time end_stamp
std_msgs/String[] ClassList
# pode ser alterado para string[]!!!!
camera_perception/BBox[] BBoxList